tsl/test/expected/bgw_telemetry.out

```diff
@@ -28,7 +28,7 @@ SELECT last_finish > :'last_finish' AS job_executed,
  WHERE job_id = :job_id;
  job_executed | last_run_success 
 --------------+------------------
- t            | t
+ t            | f
```

Comment:
is this change expected? aren't we connected with superuser/etc?

Reply:
It is a failure because telemetry is disabled in this test and it is then reported as a failure. Previous code handled this incorrectly.